selfdrive/ui/qt/maps/map_settings.cc

```diff
@@ -60,7 +60,7 @@ MapSettings::MapSettings(bool closeable, QWidget *parent) : QFrame(parent) {
   current_widget = new DestinationWidget(this);
   QObject::connect(current_widget, &DestinationWidget::actionClicked, [=]() {
     if (current_destination.empty()) return;
-    params.remove("NavDestination");
+    params.put("NavDestination", "");
```

Comment:
this is effectively a second implementation of Params::remove that just happens to make the file watcher work. can we just remove from this PR for now?